### PR TITLE
INFRA-788 Bump express version from 4.16.2 to 4.18.2

### PR DESCRIPTION
**Pull request by mdbudnick**

### CODE REVIEW

**Review by jackswisher — COMMENTED** [review]
Looks fine to me. Do we want to stick with a version of npm (pre 8) which doesn't re-write the package file or just send it on a new npm version? Maybe we specify the npm version in a `.npmrc` route if we go that way .. but this is a test service idk if we should care that much

**Comment by Robert-W** [comment]
I think the new package-lock is also backwards compatible for npm 6 and up, so we should be ok regardless

**Review by Robert-W — APPROVED** [review]

**Comment by mdbudnick** [comment]
I've added a `.nvmrc` to match what I had used

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+14.20.0
```
Comment (jackswisher) [inline .nvmrc]: missing new line

**Pull request merged**